postfix/edition15.html

```diff
@@ -60,13 +60,14 @@ <h3>
 <h6><a href="#editions">15th edition</a></h6>
 <h6>
 	Author <a href="#author">Ivar Abrahamsen</a>
+	Author <a href="#author">Daniel Hansson</a>
```

Comment:
:) 
I very much appreciate the effort but I think perhaps I will add a proper contributors section. There have been a few contributors over the years so  I'll do that in a separate commit/PR  If you have a link twitter/website/github you prefer to link to then let me know.

Reply:
Of course, if it turns out you rewrite the entire howto then maybe we need to revisit the main author part!

Reply:
Fixed!